app/(route)/ranking/page.tsx

```diff
@@ -0,0 +1,102 @@
+'use client';
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const RankingPage = () => {
+    const [additionalImagesCounts, setAdditionalImagesCounts] = useState([5, 5, 5]);
+    const [buttonClicked, setButtonClicked] = useState([false, false, false]);
+
+    const handleShowMoreImages = (index: number) => {
+        setButtonClicked((prev) => prev.map((value, i) => (i === index ? true : value))); // 더보기 클릭 시 버튼 색 변환
+        setAdditionalImagesCounts((prev) => prev.map((value, i) => (i === index ? value + 5 : value))); // 더보기 클릭 시 5개씩 추가 출력
+        setTimeout(() => {
+            setButtonClicked((prev) => prev.map((value, i) => (i === index ? false : value)));
+        }, 100);
+    };
+
+    const buttonLabels = ["남성 신발 인기 순위", "여성 신발 인기 순위", "샌들 인기 순위"];
+
+    return (
+        <div>
+            {buttonLabels.map((label, index) => (
+                <div key={index}>
+                    <h3 style={{ margin: '30px 0 5px 0' }}>{label}</h3>
```

Comment:
```suggestion
                    <h3 style={{ margin: '3rem 0 5rem' }}>{label}</h3>
```
rem으로 바꿔주세요!
이후 모든 코드도 확인해주세요

Reply:
10px = 1rem으로 해주시면 됩니다.!

Reply:
네!